### DISCUSSION TITLE
Which settings for using AG on a single pc (Win10) ?

### DISCUSSION THREAD

**Discussion by Fmikbox** [apohl79/audiogridder#834] [General]
Hi, I've just discovered Audiogridder and would like to use it on a single pc (under Win10).
I am using Cubase Pro11 as DAW and I have been teased by comments found on the web on how I could make a more efficient use of the cpu by loading "heavy" plugins, like those by Acustica Audio, through AG, even within a single computer, with no need to use a separate machine as a server.
I have downloaded and installed all the SW but it doesn't work, so I think I might have to adjust the settings of the server and/or the client.
Can anybody describe in details which settings should be used for this kind of use?
Thanks.

**Comment by CrossGridder** [comment]
With a mac or linux setup it would have be so easy. Maybe you will consider building a linux server once you discover that CrossGridder does this and much more. AG plugin and AG server on the same Linux pc? It's already available if you use CrossGridder 1.2 Check my repository

**Comment by emack999** [comment]
> I have downloaded and installed all the SW but it doesn't work

What doesn't work, exactly?

**Reply by Fmikbox** [reply]
I had installed the 1.2 beta version in first instance and the plugin was not connecting to the server.
So yesterday I uninstalled it and installed the v.1.1 and now it works. I already had some tests with Acustica Audio plugins and I had no problems at all.

**Reply by Fmikbox** [reply]
The only issue that I immediately felt was that muting and un-muting the vst (like a A-B comparison with/without the vst) is a bit curbersome, I cannot do it as I would do with the vst directly inserted on the channel.

**Comment by Jrel01** [comment]
I got the 1.2 Beta to work on my Windows 10 Pro PC, but there were some issues at first.

1. After installation and it completing the scan of my plugins, I could not save AudioGridder server settings, not with Admin privileges, and not without.  Because server settings would not save, AG would rescan my plugins every time I restarted the server.  I also could not Quit the server from the plugin tray.  Audiogridder would just restart.  I had to use AnVir Task Manager to simultaneously kill 3 processes of AudioGridderServer.exe that were in memory.  If I only killed one, another would start up automatically.

2.  I tried editing the .cfg file to add information and the .cfg file saved my edits. But when restarting the server, it would only use the default installation settings; regardless of what the .cfg file showed.

3. Through trial and error of messing with the plugin and the server settings, I discovered if I set Screen Capturing Mode to Disabled, then my server settings would save; plus I could also set it to Local mode.

4.  I can't move the initial startup/scanning logo to another monitor, it only stays on the primary monitor.

(Let me know if I should post this information in another post).
